gremlin-dotnet/test/pom.xml

```diff
@@ -161,192 +179,10 @@ limitations under the License.
-                                        <property>
-                                            <name>executionName</name>
-                                            <value>${project.name}</value>
-                                        </property>
-                                        <property>
-                                            <name>projectBaseDir</name>
-                                            <value>${project.basedir}</value>
-                                        </property>
-                                        <property>
-                                            <name>tinkerpopRootDir</name>
-                                            <value>${tinkerpop.root.dir}</value>
-                                        </property>
-                                    </properties>
-                                    <scripts>
-                                        <script>${gremlin.server.dir}/src/test/scripts/test-server-start.groovy</script>
-                                    </scripts>
-                                </configuration>
-                            </execution>
-                            <execution>
-                                <id>gremlin-server-stop</id>
-                                <phase>post-integration-test</phase>
-                                <goals>
-                                    <goal>execute</goal>
-                                </goals>
-                                <configuration>
-                                    <properties>
-                                        <property>
-                                            <name>skipTests</name>
-                                            <value>${skipTests}</value>
-                                        </property>
-                                        <property>
-                                            <name>executionName</name>
-                                            <value>${project.name}</value>
-                                        </property>
-                                    </properties>
-                                    <scripts>
-                                        <script>${gremlin.server.dir}/src/test/scripts/test-server-stop.groovy</script>
-                                    </scripts>
-                                </configuration>
-                            </execution>
                         </executions>
                     </plugin>
                 </plugins>
             </build>
         </profile>
-        <!--
-          This profile will include neo4j for purposes of transactional testing within Gremlin Server.
-          Tests that require neo4j specifically will be "ignored" if this profile is not turned on.
-        -->
-        <profile>
-            <id>include-neo4j</id>
-            <activation>
-                <activeByDefault>false</activeByDefault>
-                <property>
-                    <name>includeNeo4j</name>
```

Comment:
This property is still used in the GHA job. Please remove it there as it doesn't work any more as these tests are now enabled by default. (Or add support for this property back instead of always executing these tests, but I don't have a strong preference here about whether that's necessary or not.)

Reply:
Removed flags from .NET and Javascript GHA in [a25300b](https://github.com/apache/tinkerpop/pull/1788/commits/a25300b27cfbf949826ea810dc10059aa7e43f8d)